reverseString.js

```diff
@@ -0,0 +1,4 @@
+const reverseString = string => string.split('').reverse().join('');
```

Comment:
I sometimes have a hard time remembering the default value of `split` so I prefer to use `[...string]`, but both are fine 😊.

Reply:
Interesting, I never used the spread operator on strings 🤔